src/osd/PG.cc

```diff
@@ -5591,6 +5591,8 @@ ostream& operator<<(ostream& out, const PG& pg)
       << " " << pg.up;
   if (pg.acting != pg.up)
     out << "/" << pg.acting;
+  if (pg.is_ec_pg())
+    out << "p" << pg.get_primary();
```

Comment:
how about "p="?

Reply:
just trying not to add more characters to the prefix than we need.  p seemed more suggestive than @